### PR TITLE
`IndependentOperator` hotfix

### PR DESCRIPTION
**Pull request by yardasol**
This PR is a hotfix for `IndependentOperator` functionality. While doing more extensive testing and after extensive discussion with @paulromano , we discovered there was an implementation error: we were multiplying the fission cross sections by material volume and nuclide density in the inner normalization helper `_IndependentNormalizationHelper`, but this was already being done in `_IndependentRateHelper`, hence we were obtaining results that were way off of what they should been relative to a coupled case. 

We also found that the first-step results for fission yield nuclides were behaving as if there were no reaction rates for those nuclides, which made the first-step results very difference from the transport-coupled case. This is due to the fact that `dilute_initial` for `TransportOperator` is not user-accessible and is 0 by default.

This could have been prevented if a more robust regression test were present, so I have also added additional functionality to test results from a transport-coupled depletion calculation and a transport-independent depletion calculation over several timestep sizes.

@paulromano and I also independently discovered a bug in `Results.export_to_materials()` that was causing some errors in a validation study for transport-independent depletion. The implementation in this PR is his.

Specifically, this PR:
- Adds the `dilute_initial` parameter to `TransportOperator`
- Removes the `_IndependentRateHelper` inner class from` TransportOperator`
- Updates and overhauls the `depletion_no_transport` regression test
- Fixes a bug in `Results.export_to_materials()` that was causing some of the nuclides in the resulting `Materials` object to have different units on their compositions.
- Shoehorned in: minor docstring and docpage spelling and grammar fixes

### CODE REVIEW

File: docs/source/usersguide/depletion.rst [hunk 1]
```diff
@@ -318,7 +319,7 @@ normalizing reaction rates:
 
    .. math::
 
-      \phi = \frac{P}{V \cdot \sum_i (Q_i \cdot \Sigma^f_i \cdot \rho_i)}
+      \phi = \frac{P}{V \cdot \sum_i (Q_i \cdot \sigma^f_i \cdot \rho_i)}
```
Comment (yardasol) [inline docs/source/usersguide/depletion.rst:322]:
```suggestion
      \phi = \frac{P}{V \cdot \sum_i (Q_i \cdot \sigma^f_i \cdot \n_i)}
```

File: docs/source/pythonapi/deplete.rst [hunk 2]
```diff
@@ -43,7 +43,7 @@ algorithms <http://hdl.handle.net/1721.1/113721>`_.
     SILEQIIntegrator
 
 Each of these classes expects a "transport operator" to be passed. OpenMC
-provides The following classes implementing transpor operators:
+provides The following classes implement transport operators:
```
Comment (paulromano) [inline docs/source/pythonapi/deplete.rst:46]:
```suggestion
provides the following transport operator classes:
```

**Review by paulromano — CHANGES_REQUESTED** [review]

File: openmc/deplete/openmc_operator.py [hunk 3]
```diff
@@ -528,13 +528,13 @@ def _calculate_reaction_rates(self, source_rate):
...
                 self._normalization_helper.update(
-                    tally_rates[:, fission_ind],
-                    mat_index=mat_index)
+                    tally_rates[:, fission_ind])
 
             # Divide by total number and store
             rates[i] = self._rate_helper.divide_by_adens(number)
 
         # Scale reaction rates to obtain units of reactions/sec
+        print(f"flux, power, or source rate: {self._normalization_helper.factor(source_rate)}")
```
[the remark this answers is not in the document]
Comment (yardasol) [reply openmc/deplete/openmc_operator.py:537]: whoops, forgot to remove this!

File: openmc/deplete/microxs.py [hunk 4]
```diff
@@ -89,7 +92,12 @@ def from_model(cls,
 
         # create temporary run
         with tempfile.TemporaryDirectory() as temp_dir:
-            statepoint_path = model.run(cwd=temp_dir)
+            if run_kwargs is None:
+                run_kwargs = {}
+                model.init_lib()
+            elif 'cwd' in run_kwargs:
+                run_kwargs.pop('cwd')
+            statepoint_path = model.run(cwd=temp_dir, **run_kwargs)
```
Comment (paulromano) [inline openmc/deplete/microxs.py:100]: I wouldn't put `init_lib()` within here. If a user wants to have things run through `openmc.lib`, they can call `model.init_lib()` themselves prior to calling `MicroXS.from_model`. Also, using the `setdefault` method on a dictionary can help clean this up:
```suggestion
            run_kwargs.setdefault('cwd', temp_dir)
            statepoint_path = model.run(**run_kwargs)
```

**Review by paulromano — CHANGES_REQUESTED** [review]

**Review by paulromano — APPROVED** [review]
Thanks for the fixes @yardasol!

**Pull request merged**